src/states/Postroll.js

```diff
@@ -43,7 +43,6 @@ export default class Postroll extends AdState {
     const player = this.player;
 
     if (!player.ads.isAdPlaying() && !this.contentResuming) {
-      player.ads.debug('startLinearAdMode (Postroll)');
```

Comment:
why remove this log?

Reply:
I had added it in a relatively recent commit and found it's basically redundant with the "Starting ad break" log.